traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -630,6 +630,25 @@ <h3>Current Value</h3>
                             </aside>
                         </div>
                     </div>
+
+					<div class="form-group" ng-class="{'has-error': hasError(routingConfig.ecsEnabled), 'has-feedback': hasError(routingConfig.ecsEnabled)}">
```

Comment:
looks like you have some formatting issues going on in these tpl files.

Reply:
fixed